### PR TITLE
Drop supports Node.js v10.x and v13.x

### PR DESCRIPTION
**Pull request by ota-meshi**

### CODE REVIEW

**Comment by ota-meshi** [comment]
@RunDevelopment 
We are planning v1.0 in the next release, so I think we can merge if this PR is okay.

**Comment by RunDevelopment** [comment]
@ota-meshi We also have a few other breaking PRs. Maybe we could release one last v0.x version right now and then merge all breaking PRs? 

We had a few fixes and additions since v0.12.0, so we might as well make them available to our users before we start breaking things.

**Comment by ota-meshi** [comment]
OK. I will release 0.13 this week or next week as the last v0 release.

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ npm install --save-dev eslint eslint-plugin-regexp
 > **Requirements**
 >
 > - ESLint v6.0.0 and above
-> - Node.js v8.10.0 and above
+> - Node.js v12.x, v14.x and above
```
Comment (RunDevelopment) [inline README.md:40]: It might be nice to write that down in [`package.json`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) as well.

**Review by RunDevelopment — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ota-meshi) [reply README.md:40]: Good idea! I added it.

**Review by RunDevelopment — APPROVED** [review]

**Comment by RunDevelopment** [comment]
@ota-meshi You think we can merge this?

**Comment by ota-meshi** [comment]
@RunDevelopment Yes. I think we can merge this PR.

**Pull request merged**